scipy/stats/unuran/unuran_wrapper.pyi

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import TYPE_CHECKING, Union, Any, Callable, Tuple, Optional, List
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
+
+
+SeedType = Union[np.random.RandomState, np.random.Generator, int,
+                 npt.ArrayLike]
+
+
+__all__: List[str]
+
+
+class Method:
+    ...
+
+
+class TransformedDensityRejection(Method):
+    def __init__(self,
```

Comment:
should we create a new line for each parameter for better readability?

Reply:
> should we create a new line for each parameter for better readability?

That will be helpful. Thanks! I will push this in the next commit and we can see how it looks

Reply:
Done.